tests/test_workers/test_cleanup_job.py

```diff
@@ -201,6 +201,33 @@ def test_cleanup_job_success(mock_requests, mock_auth_requests, mock_dt):
     mock_requests.return_value.json.side_effect = [mock_complete, mock_in_progress, mock_failed]
     mock_auth_requests.return_value.ok = True
     main()
+    calls = [
+        mock.call(
+            "http://cachito.domain.local/api/v1/requests/50",
+            json={"state": "stale", "state_reason": "The request has expired"},
+            timeout=60,
+        ),
+        mock.call(
+            "http://cachito.domain.local/api/v1/requests/51",
+            json={"state": "stale", "state_reason": "The request has expired"},
+            timeout=60,
+        ),
+    ]
+    assert mock_requests.call_count == 3
+    assert mock_auth_requests.call_count == 2
+    mock_auth_requests.assert_has_calls(calls)
+
+
+@mock.patch("cachito.workers.config.Config.cachito_request_lifetime", 1)
+@mock.patch("cachito.workers.cleanup_job.datetime")
+@mock.patch("cachito.workers.cleanup_job.auth_session.patch")
+@mock.patch("cachito.workers.cleanup_job.session.get")
+def test_cleanup_job_fail(mock_requests, mock_auth_requests, mock_dt):
```

Comment:
IMO this can be merged into the previous test. For example, you could just add a second failed request to `mock_failed['items']` with an older `'updated'` value. For this to work, you would also have to change the mocked datetime object slightly, see next suggestion.